instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld

```diff
@@ -380,20 +380,24 @@
 		],
 		"ontologyIdentifier": null
 	},
-  "homepage": null,
-  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see Related publications below).",
+  "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
+  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite these publications: Papp et al., 2014 and Osen et al., 2019.",

```

Comment:
it is better to state full citations. you don't want people to search for the papers (at least state the doi)

Reply:
Full citations have been added.